src/project_generator.rs

```diff
@@ -499,6 +499,16 @@ ignore=[
 "#,
     );
 
+    if &project_info.is_pixi_project {
+        pyproject.push_str(
+            r#"[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+
+"#,
+        )
+    };
```

Comment:
Let's move this above the mypy section so build stuff is together and linting stuff is together.